LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 redhat-operator-ecosystem
```

Comment:
I might be wrong by here should be Red Hat instead of `redhat-operator-ecosystem`
I do not know any company/person `redhat-operator-ecosystem` :-)

Reply:
nice catch. updated